CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Added 
 - Add the initial Xcode project.
-- Add the Swiftlint script to the build phase of the Woof target.
-- Add the Swiftformat script to the build phase of the Woof target.
+- Add the `Swiftlint` script to the build phase of the Woof target.
+- Add the `Swiftformat` script to the build phase of the Woof target.
 - Update the deployment target to 15.0+. 
+- Add `Owner` model to the model layer.
```

Comment:
Missed:
- Create `project structure` according to MVVM architecture pattern.
or
-Add  'code's organization` by layers according to MVVM.

Reply:
Added missing PRs, thanks.